src/App.svelte

```diff
@@ -1,14 +1,17 @@
 <script lang="ts">
-	import { navigationStore } from "./stores/stores";
+	import {loadedComponentStore, navigationStore} from "./stores/stores";
 	import type {NavigationState} from "./machinery/NavigationState";
 	import Menu from "./view/Menu.svelte";
 	import Setup from "./view/Setup.svelte";
 	import About from "./view/About.svelte";
 	import Wallet from "./view/Wallet.svelte";
 	import UnlockWallet from "./view/UnlockWallet.svelte";
-	import CreateWallet from "./view/CreateWallet.svelte";
 	import SoftwareKeys from "./view/SoftwareKeys.svelte";
 	import Toast from "./components/Toast.svelte";
+	import Onboard from "./view/Onboard.svelte";
+	import {afterUpdate, onMount} from "svelte";
+	import { walletExists } from "./machinery/secure-storage";
```

Comment:
Inconsistent to the other imports. Should be `import {walletExists} from "./machinery/secure-storage";`, I guess.